### ISSUE TITLE
Change strike through on symlinks to a more subtle markup

### ISSUE THREAD

**Issue by alienbogart** [punassuming/ranger.el#201]
#### Runtime Environment

- Manjaro i3 Stable (updated)
- GNU Emacs 27.0.50
- ranger-20190412.624
- [~/.emacs.d](https://github.com/mrbig033/emacs)
- [ranger use-package](https://github.com/mrbig033/emacs/blob/master/modules/packages/misc/movements.org#ranger)
- [ranger configurations](https://github.com/mrbig033/emacs/tree/master/modules/packages/misc/ranger)

Right now this is how my symlinks look on ranger.el:
![ranger-symlinks](https://user-images.githubusercontent.com/36245777/56371628-d02b4080-61d3-11e9-9608-7dbeb53d4390.png)

The strike through is annoying, I wish to change to something less intrusive. 
I use the noctilux theme, but also tested with it disabled and there were no changes. 
How can I change that?

**Comment by punassuming** [comment]
I don't have strikethroughs at all on mine. Is there some other dired package you have installed?

**Comment by Uriziel47** [comment]
I think you are also using the package dired+? Which defines the face 'diredp-omit-file-name' for files matching the regexp in 'diredp-omit-files-regexp' ( dotfiles as default ).

**Comment by punassuming** [comment]
Yes, I am trying to utilize existing dired packages. The theme controls the strike through. You should be able to customize this face in particular to address your needs.
